demos/layout-grid.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
```

Comment:
It looks like support broke when I added the CSS Grid PE, because Edge supports the old style CSS Grid, and autoprefixer is converting to that. Added the `grid: false` option to autoprefixer to prevent that. There is a [bug in autoprefixer](https://github.com/postcss/autoprefixer/issues/778) that requires an extra workaround for now, though.

Also, added the check for custom properties to hide the controls and display a warning. Unfortunately, MS Edge misreports its support for custom properties, and I can't find a reliable way of feature-detecting it, so I added a warning that gets shown if the user's browser claims it supports custom properties.